class.py

```diff
@@ -0,0 +1,34 @@
+class Employee:
+    def __init__(self, firstname, lastname, position, salary):
+        self.firstname = firstname
+        self.lastname = lastname
+        self.position = position
+        self.salary = salary
+
+    def __repr__(self):
+        return  f"First name: {self.firstname}\nLast name: {self.lastname}\nPosition: {self.position}\nSalary: {self.salary}"
+
+employee1 = Employee ('Marko', "Markić", "accountant", 6000)
+employee2 = Employee ('Hrvoje', "Horvat", "accountant", 8000)
+employee3 = Employee ('Ivana', "Ivić", "accountant", 19000)
+
+class Company:
+    def __init__(self, name, address, employees=[]):
+        self.name = name
+        self.address = address
+        self.employees = employees
+
+    def add_employee (self, employee):
+        try:
+            if employee.salary>5000 and employee.salary<20000:
+                self.employees.append(employee)     
+        except:
+            print('Could not add new employee; employee salary is less than 5000 or greater than 20000')
```

Comment:
Ovo je dobra ideja, samo ti fali da python zapravo i digne Exception. Znaci unutar `try` grane treba dodati `else` u kojem se podize Exception, a onda `except` grana hvata Exception i radi sto vec zelis, u tvom slucaju printa upozorenje. Mozes definirati sama koji Exception ce se podizati (ili cak sama definirati novi tip exception-a), ali ovdje mozes javljati npr. `ValueError` (strogo gledano to vjerojatno nije najcisce rjesenje jer se zapravo ne desava pravi `ValueError`, al ok...).

Vidi https://www.programiz.com/python-programming/user-defined-exception

Reply:
done